Backpack/src/main/java/net/skyscanner/backpack/snackbar/BpkSnackbar.kt

```diff
@@ -19,20 +20,24 @@ import net.skyscanner.backpack.util.BpkTheme
 import net.skyscanner.backpack.util.use
 
 /**
- * A builder wrapper around [Snackbar] providing the required customization.
+ * A wrapper around [Snackbar] providing the required customization.
  *
- * [BpkSnackbar.builder] creates a new builder. It's strongly recommended to set text and action
- * before you build a [Snackbar] using [BpkSnackbar.build] method.
+ * Use [BpkSnackbar.make] to create a new instance, then use it in the same way as a [Snackbar].
  *
- * @see BpkSnackbar.builder
+ * @see BpkSnackbar.make
  * @see BpkSnackbar.setText
  * @see BpkSnackbar.setAction
  * @see BpkSnackbar.setDuration
- * @see BpkSnackbar.build
+ * @see BpkSnackbar.setDuration
```

Comment:
nit: We don't need to list all methods here, this will be visible in the docs page anyway. I use @see generally to link to another related class, if any. For example you could link the length consts here.